c/include/nnstreamer-tizen-internal.h

```diff
@@ -19,6 +19,8 @@
 extern "C" {
 #endif /* __cplusplus */
 
+#define ML_NNFW_TYPE_FLARE 23 /**< FLARE framework */
```

Comment:
You should write a note in the comment describing why this is defined here, not in ```ml-api-common.h```, and when/under-which-codition this  will be migrated to there. Otherwise, months later, we won't know why this is here and we won't move it to the proper position.

Plus, as long as Tizen-next is already submited, can you move it to the proper file and prepare ACR for 2026-M1?